.github/workflows/build.yaml

```diff
@@ -0,0 +1,205 @@
+        with:
+          useLockFile: false
+
+      - name: Kick yarn
+        run: yarn install
+
+      - name: Setup Ruby 
+        uses: actions/setup-ruby@v1
+        env:
+          BUNDLE_GEMFILE: "${{ github.workspace }}/ios/Gemfile"
+        with:
+          ruby-version: '2.6'
+          bundler-cache: true
+
+      - name: Printenv
+        run: printenv && npx react-native info
+
+      - name: Android Jetify
+        run: npx jetify
+#
+      - name: Android Build
+        run: cd android && . .env.default && fastlane internal
+
+  ios-build:
+    name: iOS Build
+    runs-on: macos-latest 
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [ 14.x ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Generate .env file
+        uses: project-vagabond/template-action@v1
+        with:
+          template: .templates/.env.template
+          output: .env
+          vars: |
+            {
+              "API_HOST": "${{ secrets.API_HOST }}",
+              "SAFETYNET_KEY": "${{ secrets.SAFETYNET_KEY }}"
+            }
+
+      - name: Generate iOS AuthKey
+        run: echo "${{ secrets.IOS_AUTH_KEY }}" > "ios/AuthKey_${{ secrets.IOS_AUTH_KEY_ID }}.p8"
+
+      - name: Generate iOS Signing Key
+        run: echo "${{ secrets.SIGNING_KEY_P12_B64 }}" | base64 -d > "ios/NY COVID Proximity.key.p12"
+
+      - name: Generate iOS mobile provisioning profile        
+        run: |
+          echo "${{ secrets.IOS_MOBILE_PROV }}" | base64 -d -o "ios/COVID_Proximity_Tracing.mobileprovision"
+          uuid=`grep UUID -A1 -a "ios/COVID_Proximity_Tracing.mobileprovision" | grep -io "[-A-F0-9]\{36\}"`
+          mkdir -p ~/Library/MobileDevice/Provisioning\ Profiles
+          cp "ios/COVID_Proximity_Tracing.mobileprovision"  ~/Library/MobileDevice/Provisioning\ Profiles/$uuid.mobileprovision
+          chmod a+rw ~/Library/MobileDevice/Provisioning\ Profiles/$uuid.mobileprovision
+          echo "~/Library"
+          ls -la ~/Library
+          echo "~/Library/MobileDevice"
+          ls -la ~/Library/MobileDevice
+          echo "~/Library/MobileDevice/Provisioning Profiles/"
+          ls -la ~/Library/MobileDevice/Provisioning\ Profiles/
+
+      - name: Generate iOS firebase config DEV
+        run: echo "${{ secrets.FIREBASE_CFG_IOS_DEV }}" | base64 -d -o "ios/firebase/development/GoogleService-Info.plist"        
+
+      - name: Generate iOS firebase config PROD
+        run: echo "${{ secrets.FIREBASE_CFG_IOS_PROD }}" | base64 -d -o "ios/firebase/production/GoogleService-Info.plist"        
+            
+
+      - name: Generate iOS env.default
+        uses: project-vagabond/template-action@v1
+        with:
+          template: .templates/ios/env.default.template
+          output: ios/.env.default
+          vars: |
+            {
+              "GITHUB_WORKSPACE": "${{ github.workspace }}",
+              "APPLE_ID": "${{ secrets.FATLANE_USER }}",
```

Comment:
Typo

Reply:
That guy has probably been killing me :(